stout/flags.h

```diff
@@ -107,6 +114,20 @@ class Parser {
   // Optional for including all environment variables for parsing
   // with specific prefix.
   std::optional<std::string> environment_variable_prefix_;
+
+  // Helper for saving the current message for which subcommand flags are
+  // being parsed.
+  std::optional<google::protobuf::Message*> cur_subcommand_message_;
+
+  // Helper for saving the previous message for which subcommand flags are
+  // being parsed. It's always top message at the beginning.
```

Comment:
If it's always the top message should we change it's name to just be that?

Reply:
It's not always the top message (e.g.: there are nested subcommands)! Anyway in the [PR](https://github.com/3rdparty/stout-flags/pull/18) for parsing positional arguments I've changed the name to `cur_message_`.